### PR TITLE
Kdb rewrite meta(get, ls, rm, set, show)

### PR DESCRIPTION
**Pull request by hannes99**
Relevant files for review:
- `/src/tools/kdb/meta.c`
- `/src/tools/kdb/meta-get.c`
- `/src/tools/kdb/meta-ls.c`
- `/src/tools/kdb/meta-rm.c`
- `/src/tools/kdb/meta-set.c`
- `/src/tools/kdb/meta-show.c`

other changes are mostly doc updates (stuff like `meta-get` -> `meta get`)

## Basics

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which contains `_(my name)_`)
      **Please always add them to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - add a line in `doc/news/_preparation_next_release.md`
  - reformat the code with `scripts/dev/reformat-all`
  - make all unit tests pass
  - fix all memleaks
  - fix the CI itself (or rebase if already fixed)
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation (see [Documentation Guidelines](https://www.libelektra.org/devgettingstarted/documentation))
- [ ] I fixed all affected decisions (see [Decision Process](https://www.libelektra.org/decisions/decision-process))
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://www.libelektra.org/devgettingstarted/coding))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if everything is done** and no further pushes are planned by you.

### CODE REVIEW

**Comment by hannes99** [comment]
jenkins build libelektra please

File: src/tools/kdb/meta-get.c [hunk 1]
```diff
@@ -29,7 +32,59 @@ void addMetaGetSpec (KeySet * spec)
...
+	{
+		RETURN (2)
+	}
+
+	const char * metaName = GET_OPTION (options, "metaname");
+
+	KeySet * conf = ksNew (0, KS_END);
+	KDB * handle = kdbOpen (NULL, errorKey);
+
+	Key * cascadingParent = keyNew ("/", KEY_END);
```
Comment (kodebach) [inline src/tools/kdb/meta-get.c:51]: Why this parent? This probably loads far more data than necessary.

It's also inconsistent with all the other `kdb meta` commands and especially `kdb meta show`, which I'd expect to print the same value for a given meta name, so both should be able to use the same parent.

**Review by kodebach — CHANGES_REQUESTED** [review]
LGTM, but `kdb meta get` is inconsistent with the other commands in terms of parent key. 

In general, the parent key seems to be handled differently everywhere, but this one is the most problematic.

File: src/tools/kdb/meta-get.c (hunk 1 above)
Comment (hannes99) [reply src/tools/kdb/meta-get.c]: Good question, sometimes past hannes does weird stuff, fixed it :)

**Comment by hannes99** [comment]
jenkins build libelektra please

**Review by kodebach — APPROVED** [review]
LGTM

**Pull request closed without merging**